src/basic/chase-symlinks.h

```diff
@@ -24,6 +24,7 @@ typedef enum ChaseSymlinksFlags {
                                              * full path is still stored in ret_path and only the returned
                                              * file descriptor will point to the parent directory. */
         CHASE_MKDIR_0755         = 1 << 11, /* Create any missing parent directories in the given path. */
+        CHASE_FILENAME           = 1 << 12, /* Only return the last component of the resolved path */
```

Comment:
call this CHASE_RETURN_FILENAME? or CHASE_EXTRACT_FILENAME?

because otherwise one might think the filename is what is chased here, but it it is not, it's what is returned. i.e. it's not input put output of the function